pkg/domain/infra/abi/manifest.go

```diff
@@ -164,27 +164,29 @@ func (ir *ImageEngine) remoteManifestInspect(ctx context.Context, name string, o
 		return nil, latestErr
 	}
 
-	switch manType {
-	case manifest.DockerV2Schema2MediaType:
-		logrus.Warnf("The manifest type %s is not a manifest list but a single image.", manType)
-		schema2Manifest, err := manifest.Schema2FromManifest(result)
-		if err != nil {
-			return nil, fmt.Errorf("parsing manifest blob %q as a %q: %w", string(result), manType, err)
-		}
-		if result, err = schema2Manifest.Serialize(); err != nil {
-			return nil, err
-		}
-	default:
-		listBlob, err := manifest.ListFromBlob(result, manType)
-		if err != nil {
-			return nil, fmt.Errorf("parsing manifest blob %q as a %q: %w", string(result), manType, err)
-		}
-		list, err := listBlob.ConvertToMIMEType(manifest.DockerV2ListMediaType)
-		if err != nil {
-			return nil, err
-		}
-		if result, err = list.Serialize(); err != nil {
-			return nil, err
+	if opts.Format != "oci" {
```

Comment:
If I understand this correctly, the semantics `if false` is not “OCI” but “raw without conversion”. Why name the format value OCI, then?

Reply:
Not using this, instead using `skopeo` in test.